include/entry.h

```diff
@@ -176,6 +176,10 @@ class Entry
 
   private:
     zim::Entry entry;
+
+  private:
+    Entry(zim::Entry entry, bool marker);
```

Comment:
Explain in a comment the purpose of this constructor

Reply:
Done